formfyxer/lit_explorer.py

```diff
@@ -972,6 +972,39 @@ def get_citations(text: str, tokenized_sentences: List[str]) -> List[str]:
     return citations_with_context
 
 
+def get_sensitive_fields(fields: List[str]) -> List[str]:
+    """
+    Given a list of fields, identify those related to sensitive information. Sensitive fields include 
+    Social Security Number(SSN)), Driver's License (DL), and account numbers.
+    """
+    # NOTE: omitting CID since it has a lot of false positives.
```

Comment:
I'm guessing it means credit card ID (verification code printed on the card in the signature line?)

Reply:
Yeah, it's Credit Card ID in this context. I was seeing a lot of CID matches in the test forms and it's not a common way to reference credit card info, so I left it out.

Reply:
Gotcha. That makes sense. This would make that a bit clearer:

```suggestion
    # NOTE: omitting "CID" for Credit Card IDs since it has a lot of false positives.
```

Reply:
Updated.